tests/simplex/matrix_free_02.cc

```diff
@@ -45,118 +45,10 @@
 #include <deal.II/simplex/grid_generator.h>
 #include <deal.II/simplex/quadrature_lib.h>
 
-#include "../tests.h"
+#include "./tests.h"
```

Comment:
Let us not call the file `tests.h` but something else, say `simplex_grids.h` or something related. Assuming a different one from the base directory `tests.h` will be confusing for anyone adding tests here.

Reply:
OK. I have renamed the file.